common/src/main/resources/assets/voicechat/lang/en_us.json

```diff
@@ -1,217 +1,212 @@
 {
   "key.categories.voicechat": "Voice Chat",
-  "key.push_to_talk": "Push To Talk",
+  "key.push_to_talk": "Push to Talk",
   "key.whisper": "Whisper",
   "key.mute_microphone": "Mute Microphone",
-  "key.disable_voice_chat": "Disable Voice Chat",
+  "key.disable_voice_chat": "Disable Voice Chat Sounds",
   "key.hide_icons": "Hide Voice Chat Icons",
   "key.voice_chat_settings": "Voice Chat Settings",
-  "key.voice_chat": "Voice Chat GUI",
-  "key.voice_chat_group": "Voice Chat Group",
+  "key.voice_chat": "Voice Chat Menu",
+  "key.voice_chat_group": "Group Management",
   "key.voice_chat_toggle_recording": "Toggle Recording",
   "key.voice_chat_adjust_volumes": "Adjust Volumes",
   "gui.voicechat.voice_chat.title": "Voice Chat",
   "gui.voicechat.voice_chat_settings.title": "Voice Chat Settings",
   "gui.voicechat.adjust_volume.title": "Adjust Volumes",
-  "gui.voicechat.select_microphone.title": "Select Microphone",
-  "gui.voicechat.select_speaker.title": "Select Speaker",
-  "gui.voicechat.group.title": "Group Chat",
-  "gui.voicechat.join_create_group.title": "Join or Create Group",
+  "gui.voicechat.select_microphone.title": "Select a Microphone",
+  "gui.voicechat.select_speaker.title": "Select Speakers",
+  "gui.voicechat.group.title": "Group Management",
+  "gui.voicechat.join_create_group.title": "Join or Create a Group",
   "gui.voicechat.create_group.title": "Create a Group",
   "gui.voicechat.enter_password.title": "Enter Group Password",
   "message.voicechat.settings": "Settings",
   "message.voicechat.group": "Group",
-  "message.voicechat.voice_chat_volume": "Voice chat volume: %s",
-  "message.voicechat.microphone_amplification": "Microphone amplification: %s",
-  "message.voicechat.mic_test_unavailable": "Microphone testing unavailable",
-  "message.voicechat.mic_test.disabled": "Press to enable microphone testing",
-  "message.voicechat.mic_test.enabled": "Press to disable microphone testing",
+  "message.voicechat.voice_chat_volume": "Voice Chat Volume: %s",
```

Comment:
First, let me quote something here that's related:
> I generally prefer to only capitalize text in titles, not buttons, etc.

Personally, I'm also not a fan of title case capitalization, even if it is only used in titles. I would prefer everything to use sentence case capitalization because it is simply easier to read and feels more natural.

Whether, where, and when to use the title case, however, seems to be a matter of debate. For instance, Google in the latest [Material You / Material Design 3 recommends using sentence case](https://m3.material.io/foundations/content-design/style-guide/ux-writing-best-practices) for button labels, whereas [Apple](https://developer.apple.com/design/human-interface-guidelines) and [GNOME](https://developer.gnome.org/hig/guidelines/writing-style.html) in their Human Interface Guidelines suggest using title case instead.

Mojang seems to prefer the latter. That's their decision, and there's nothing I can do about it :( Therefore, in my opinion, consistency across the entire game interface, between modded and unmodded, is more important than my personal preferences. And the only way to ensure this is to follow the standards and conventions set by Mojang itself. That's why you may have noticed that many of the changes here are for "parity with vanilla Minecraft".

(As a side note, I should add that you use title case capitalization for the key binds from this mod, even though they aren't titles.)

Reply:
Yeah I think doing it like minecraft does would be the most consistent.

> (As a side note, I should add that you use title case capitalization for the key binds from this mod, even though they aren't titles.)

If mojang doesn't do that, we shouldn't do that either, thats true

Reply:
Alright, in that case, I'll leave the button labels capitalized for parity with vanilla Minecraft.

In my previous comment, I also raised the "issue" of using title case capitalization for the key binds from this mod. I realize I wasn't clear enough, and as a result, you misunderstood me, for which I apologize. Mojang's preference for capitalizing the key bind descriptions is exactly the same as for button labels. That is, title case capitalization should be used here as well, which is what you have been doing already. There's nothing to change here.

Reply:
Alright

Reply:
From the perspective of a resource pack creator (which I'm not), a slider has just always been a button, only cut and shaped in a way that makes it look like a slider (at least up until the resource pack format 16 / Minecraft 1.20.2, but let's not get into details).

That's why I completely forgot about it and haven't mentioned it in my previous comment when I should have. My bad.

Mojang seems to treat them the same when it comes to capitalizing their labels or descriptions in tooltips, so I don't see any reason to do otherwise.

Reply:
Yeah. Makes sense.